pkg/assume/assume.go

```diff
@@ -195,7 +195,7 @@ func AssumeCommand(c *cli.Context) error {
 		sessionExpiration := ""
 		green := color.New(color.FgGreen)
 		if creds.CanExpire {
-			sessionExpiration = creds.Expires.Format(time.RFC3339)
+			sessionExpiration = creds.Expires.Local().Format(time.RFC3339)
 			if os.Getenv("GRANTED_QUIET") != "true" {
 				green.Fprintf(color.Error, "\n[%s](%s) session credentials will expire %s\n", profile.Name, region, creds.Expires.Local().String())
```

Comment:
Should this use `sessionExpiration` instead of `creds.Expires.Local().String()` for consistency/simplicity?

Reply:
Thanks for opening the PR! LGTM. If you want to make this change for consistency, we can merge it in.